snmp/check.py

```diff
@@ -492,6 +492,7 @@ def submit_metric(self, name, snmp_value, forced_type, tags=[]):
             return
 
         metric_name = self.normalize(name, prefix="snmp")
+        self.log.debug("Submitting %s", metric_name)
```

Comment:
It is quite verbose I agree, but it's also the kind of thing that helps us debug specific OIDs not being collected. Anyways, it's just a single line, so if necessary we can ask them to add something if necessary.